runtime/lua/vim/lsp.lua

```diff
@@ -623,7 +623,9 @@ function lsp.enable(name, enable)
 
   -- Ensure any pre-existing buffers start/stop their LSP clients.
   if enable ~= false then
-    vim.api.nvim_command('doautoall nvim.lsp.enable FileType')
+    if vim.v.vim_did_enter == 1 then
+      vim.api.nvim_command('doautoall nvim.lsp.enable FileType')
```

Comment:
I didn't see when this got added.


```suggestion
      vim.cmd.doautoall('nvim.lsp.enable FileType')
```

Reply:
I came from #33702. Yes, I would like you to review it as well

Reply:
> Actually if possible it would be better to use nvim_exec_autocmds.

Not sure if nvim_exec_autocmds even support `doautoall`, but that's not my priority. I just want to fix this bug because it is really annoying, it throws error just after I open Nvim.

Or you can revert that PR